### ISSUE TITLE
Object prototype may only be an Object or null: undefined (After update from angular 9 to 10)

### ISSUE THREAD

**Issue by andzejsw** [angular/angular#38282]
Only on production build i have error `Object prototype may only be an Object or null: undefined`. It isn't showing in dev build. This error appeared on prod build only after updating project from Angular 9 to 10.

**Comment by AndrewKushnir** [comment]
Thanks for reporting this issue. However, you didn't provide sufficient information for us to understand and reproduce the problem. Please check out [our submission guidelines](https://github.com/angular/angular/blob/master/CONTRIBUTING.md#-submitting-an-issue) to understand why we can't act on issues that are lacking important information.

You could probably debug this problem further by removing components from the app and detect the one that is causing this issue. Once you identify the problem component/module, try to simplify it (by removing its dependencies) to see which one might cause the issue.

If the problem still persists, please file a new issue and ensure you provide all of the required information when filling out the issue template.
